awscli/customizations/s3/results.py

```diff
@@ -57,6 +59,8 @@ def _create_new_result_cls(name, extra_fields=None, base_cls=BaseResult):
 
 ErrorResult = namedtuple('ErrorResult', ['exception'])
 
+CntrlCResult = _create_new_result_cls('CntrlCResult', base_cls=ErrorResult)
```

Comment:
Small detail, but the error messages use "Ctrl" which I think is a more common way to shorten "control".  I'd prefer to either use CtrlCResult or just the full out "ControlCResult"